x-pack/metricbeat/module/sql/_meta/docs.asciidoc

```diff
@@ -895,3 +897,49 @@ The username and password to connect to the database can be provided as values t
   - query: SELECT METRIC_NAME, VALUE FROM V$SYSMETRIC WHERE GROUP_ID = 2 and METRIC_NAME LIKE '%'
     response_format: variables 
 ----
+
+=== SSL Setup
+
+The SSL configuration is driver-specific. Different drivers interpret parameters not in the same way. Subset of the https://www.elastic.co/docs/reference/beats/metricbeat/configuration-ssl#ssl-client-config[params] is supported.
+
+Currently, there are two ways to make SSL connections to the databases:
+
+- Keep `ssl.enabled` flag disabled and supply all SSL parameters in the connection string in `hosts`.
+
+- Enable `ssl.enabled` and use `ssl.*` configuration parameters.
+
+==== Limitations
+
+The module supports SSL with `mysql`, `mssql`, and `postgres` drivers. 
+
+When SSL is enabled with `ssl.enabled` parameter, only URL-formatted connection strings are accepted, like `"postgres://myuser:mypassword@localhost:5432/mydb"`, not like `"user=myuser password=mypassword dbname=mydb"`.
+
+===== `mysql` driver
+
+Params supported: `ssl.enabled`, `ssl.verification_mode`, `ssl.certificate`, `ssl.key`, `ssl.certificate_authorities`.
+
+The certificates can be passed both as file paths and as certificate content (https://www.elastic.co/docs/reference/beats/metricbeat/configuration-ssl#client-certificate-authorities[embedding certificate example]).
+
+===== `postgres` driver
+
+Params supported: `ssl.enabled`, `ssl.verification_mode`, `ssl.certificate`, `ssl.key`, `ssl.certificate_authorities`.
+
+Only one certificate can be passed to `ssl.certificate_authorities` parameter.
```

Comment:
Does postgresql otherwise support only 1 CA. Is this our limitation as we take cert path or postgresql's ?

Reply:
So if they give a combined_cert.pem path in the ssl.certificate_authorities, we parse it, right ?